src/org/zaproxy/zap/extension/ascanrules/TestCrossSiteScriptV2.java

```diff
@@ -135,6 +136,9 @@ public void init() {
     	
     @Override
     public void scan(HttpMessage msg, String param, String value) {
+        if (HttpRequestHeader.PUT.equals(msg.getRequestHeader().getMethod())) {

```

Comment:
We could just scan PUTs on a LOW threshold? But I'm ok also not scanning them at all.